tee-worker/litentry/core/assertion-build/src/achainable/achainable_amounts.rs

```diff
@@ -0,0 +1,88 @@
+// Copyright 2020-2023 Litentry Technologies GmbH.
+// This file is part of Litentry.
+//
+// Litentry is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Litentry is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Litentry.  If not, see <https://www.gnu.org/licenses/>.
+
+#[cfg(all(feature = "std", feature = "sgx"))]
+compile_error!("feature \"std\" and feature \"sgx\" cannot be enabled at the same time");
+
+#[cfg(all(not(feature = "std"), feature = "sgx"))]
+extern crate sgx_tstd as std;
+
+use crate::{achainable::request_achainable, *};
+use lc_data_providers::{
+	achainable::{Params, ParamsBasicTypeWithAmounts},
+	vec_to_string,
+};
+
+const VC_SUBJECT_DESCRIPTION: &str = "Class of year";
+const VC_SUBJECT_TYPE: &str = "ETH Class of year Assertion";
```

Comment:
This is confusing IMO - this VC is about the "class of year", but it's named `amounts.rs` :(

I can understand it's using the `Amounts` API from achainable, but still...